code/game/objects/items/weapons/teleportation.dm

```diff
@@ -139,7 +139,7 @@ Frequency:
 		return
 	var/T = L[t1]
 	user.show_message("<span class='notice'>Locked In.</span>", 2)
-	var/obj/effect/portal/P = new /obj/effect/portal(get_turf(src), T, src)
+	var/obj/effect/portal/P = new /obj/effect/portal(get_turf(src), T, src, user)
```

Comment:
Need a named argument because `user` is currently where the `lifespan` arg is.
```suggestion
	var/obj/effect/portal/P = new /obj/effect/portal(get_turf(src), T, src, creation_mob = user)
```
Either that or `..., src, 300, user`